javascript-functions/main.js

```diff
@@ -0,0 +1,43 @@
+function convertMinutesToSeconds(minutes, seconds) {
+  var multiply = minutes * seconds;
+  return multiply;
+}
+
+var convertMinutesToSecondsResult =
+convertMinutesToSeconds(5, 60);
+console.log('convertMinutesToSecondsResult:', convertMinutesToSecondsResult);
+
+function name(Hey, Beavis) {
+  var name = 'Hey, Beavis';
+  return name;
+}
+
+var nameResult = name('Beavis');
```

Comment:
How about `name('Ryan')` or `name('Katherine')`? Those are not returning `Hey, Ryan` and `Hey, Katherine`. You need to build a function that returns a dynamic value based on the parameters it gets